src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Extentions/HostBuilderExtentions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NLog.Extensions.Logging;
+using Serilog;
+using System;
+
+namespace Exadel.CrazyPrice.Common.Extentions
+{
+    public static class HostBuilderExtentions
+    {
+        public static IHostBuilder SetupLogger(this IHostBuilder hostBuilder)
+        {
+            return hostBuilder.ConfigureLogging((hostingContext, logging) =>
+            {
+                logging.ClearProviders();
+                logging.AddConfiguration(hostingContext.Configuration.GetSection("Logging"));
+
+                if (HasLog(hostingContext, "LogToSerilog"))
+                {
+                    Environment.SetEnvironmentVariable("BASEDIR", AppDomain.CurrentDomain.BaseDirectory);
+                    logging.AddSerilog(new LoggerConfiguration()
+                        .ReadFrom.Configuration(hostingContext.Configuration)
+                        .CreateLogger());
+                }
+
+                if (HasLog(hostingContext, "LogToNLog"))
+                {
+                    logging.AddNLog();
```

Comment:
Does NLog standart parameters with nlog.config used here?

Reply:
logging.AddNLog() changed to logging.AddNLogWeb() for support template like ${aspnet-request-method}